### DISCUSSION TITLE
Trajectory tracking/Point-to-Point moiton

### DISCUSSION THREAD

**Discussion by LeonardoHou** [do-mpc/do-mpc#395] [Q&A]
I want to use this module to realize a "Trajectory tracking/Point-to-Point motion" model predictive control scheme, but do not found the relevant API, is there anyone who can give me some tips about this?

**Comment by 4flixt** [comment]
Hi @LeonardoHou,

your application should be possible to realize with do-mpc. You will need to use time-varying parameters for trajectory-tracking which is explained [here](https://www.do-mpc.com/en/latest/FAQ.html#time-varying-parameters). An example for a trajectory tracking problem is given [here](https://github.com/do-mpc/do-mpc/blob/master/examples/rotating_oscillating_masses_mhe_mpc/template_mpc.py).

Best,
4flixt
